### PR TITLE
Explicitly call "geodata" in path (#25)

### PR DESCRIPTION
**Pull request by philboeselager**

### CODE REVIEW

File: conf/routes [hunk 1]
```diff
@@ -1,19 +1,19 @@
-GET     /                                 controllers.Application.index()
+GET     /geodata/search                           controllers.Application.index()
```
Comment (fsteeg) [inline conf/routes]: The index page should be at just `/geodata`.

See prefix section at http://hbz.github.io/#data-module-specification

**Comment by fsteeg** [comment]
Before we deploy this, we need to make the required changes in lobid-organisations, where the routes are called for the transformation. Which made me realize we need a separate staging system for geodata. Opened https://github.com/hbz/geodata/issues/34. Ping #25.

**Comment by fsteeg** [comment]
Blocked by https://github.com/hbz/geodata/issues/34, which is blocked by https://github.com/hbz/geodata/issues/23, which is `ready` for work. Unassigning and moving to backlog. Note that after https://github.com/hbz/geodata/issues/34 and https://github.com/hbz/geodata/issues/23 are resolved, this still needs a coordinated pull request against lobid-organisations, see https://github.com/hbz/geodata/pull/33#issuecomment-170910260.

**Comment by SBRitter** [comment]
Merged

**Comment by SBRitter** [comment]
Solves #25

**Pull request closed without merging**